app/models/subject.rb

```diff
@@ -1,7 +1,20 @@
 class Subject < ApplicationRecord
   has_many :notifications, foreign_key: :subject_url, primary_key: :url
 
+  BOT_AUTHOR_REGEX = /\A(.*)\[bot\]\z/
+  private_constant :BOT_AUTHOR_REGEX
+
   def author_url
-    "#{Octobox.config.github_domain}/#{author}"
+    if bot_author?
+      "https://github.com/apps/#{BOT_AUTHOR_REGEX.match(author)[1]}"
```

Comment:
* This assumes that bot usernames directly match GitHub app names every time. I don't know if that's actually true or not.
* Hardcoded here - how does this affect GitHub Enterprise, etc? Can you get apps from other places? I don't know if this assumption is correct either.

@tarebyte Can you confirm any of this?

Reply:
@chrisarcand at the moment GitHub Apps aren’t available in Enterprise. When it does ship it should point to their own instance, so let’s use configured domain for now.

Reply:
Excellent, thanks.